### PR TITLE
Add WCAG autocomplete attributes to FormConfig

### PR DESCRIPTION
**Pull request by jakerarr**
- Add autocomplete constants file
- Update FormField with get/set for autoComplete attribute
- Update twig files to add autocomplete attribute when applicable

### CODE REVIEW

**Review by GameCharmer — CHANGES_REQUESTED** [review]
Skip this one, move onto the below changes

File: src/Core/AutoCompleteConstants.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
+<?php
+namespace GCWorld\FormConfig\Core;
```
Comment (GameCharmer) [inline src/Core/AutoCompleteConstants.php]: Move this to Interfaces.  It's 5 folders down. :P

File: src/Forms/FormField.php [hunk 2]
```diff
@@ -227,6 +233,27 @@ public function getNameRaw()
...
+    }
+
+    /**
+     * @param $component
+     * @return $this
+     * @throws \Exception
+     */
+    public function setAutoComplete($component)
+    {
+        if (!isset(AutoCompleteConstants::COMPONENTS[$component])) {
```
Comment (GameCharmer) [inline src/Forms/FormField.php]: If you're going to force exceptions, you need to make sure ALL possible options exist, like "on" and "off"

File: twig/BS3/field_parents/select2HTML.twig [hunk 3]
```diff
@@ -1,5 +1,11 @@
 {% macro render(field, multi) %}
-    <input type="hidden" name="{{ field.name }}" id="{{ field.id }}" class="{{ field.class }}" {{ field.getDataAttributeString()|raw }}>
+    <input type="hidden"
+           name="{{ field.name }}"
+           id="{{ field.id }}"
+           class="{{ field.class }}"
+           {{ (field.getAutoComplete() = '' ? 'autocomplete="off"' : 'autocomplete="' ~ field.getAutoComplete() ~ '"') }}
```
Comment (GameCharmer) [inline twig/BS3/field_parents/select2HTML.twig]: For every one of these that you're changing, make sure you're changing the accompanying PHP file in src/Fields to ensure the method exists.

You may want to add your autoComplete method to a Trait (in the traits folder), then USE it in FormField and in any other fields that will have the autoComplete method

File: src/Core/AutoCompleteConstants.php (hunk 1 above)
Comment (jakerarr) [reply src/Core/AutoCompleteConstants.php]: Moved to interfaces!

File: src/Forms/FormField.php (hunk 2 above)
Comment (jakerarr) [reply src/Forms/FormField.php]: Added ON/OFF options!

File: twig/BS3/field_parents/select2HTML.twig (hunk 3 above)
Comment (jakerarr) [reply twig/BS3/field_parents/select2HTML.twig]: Added changes to PHP files.  Made trait and applied it to FormField and all php files where the associated twig file was changed!

**Review by GameCharmer — APPROVED** [review]

**Pull request merged**